tests/Validation/Constraints/IntConstraintsTest.php

```diff
@@ -1,87 +0,0 @@
-<?php
-
-/* Copyright (c) 2017 Stefan Hecken <stefan.hecken@concepts-and-training.de> Extended GPL, see docs/LICENSE */
-
-use ILIAS\Validation;
-use ILIAS\Data;
-
-/**
- * TestCase for the factory of constraints
- *
- * @author Stefan Hecken <stefan.hecken@concepts-and-training.de>
- */
-class IntConstraintsTest extends PHPUnit_Framework_TestCase {
```

Comment:
Because almost all of this code can be reused for non-int tests as well (see isNumeric). Just renamed it to standard and added just a slight bit of more flexibility (to process mulitple ok's and errors at once).

Reply:
Ah, thx!